package.json

```diff
@@ -48,13 +50,24 @@
     "@types/react": "^16.13.1",
     "@types/react-helmet": "^6.1.6",
     "@types/styled-components": "^5.1.26",
+    "@typescript-eslint/eslint-plugin": "^6.2.0",
+    "@typescript-eslint/parser": "^6.2.0",
     "babel-jest": "^29.5.0",
     "babel-loader": "^9.1.2",
     "csstype": "^3.1.1",
+    "eslint": "^8.45.0",
+    "eslint-config-standard-with-typescript": "^37.0.0",
+    "eslint-plugin-import": "^2.25.2",
```

Comment:
Question; so what plugins have we added here, what does each give us?

What other plugins have been considered? For example. did you look at any of the following?

- eslint-plugin-react-hooks
- eslint-plugin-jsx-a11y
- eslint-plugin-styled-components-a11y
- eslint-plugin-prettier
- eslint-plugin-storybook
- eslint-plugin-testing-library
- eslint-plugin-jest
- eslint-plugin-unicorn
- eslint-plugin-security
- eslint-plugin-react-perf

Reply:
"eslint-config-standard-with-typescript": "^37.0.0"
This package provides an ESLint configuration that includes rules for the Standard style guide along with support for TypeScript-specific rules. It ensures that your TypeScript code adheres to the Standard style guide conventions.

"eslint-plugin-import": "^2.25.2"
This plugin provides ESLint rules for linting and enforcing best practices related to import statements in your code. It helps ensure consistent and correct import usage, such as detecting unused imports, enforcing specific import order, and preventing circular dependencies.

"eslint-plugin-n": "^15.0.0 || ^16.0.0"
This plugin helps you enforce rules based on the Node.js version you are targeting. It allows you to define rules that are specific to a particular Node.js version or range. For example, you might have different APIs available in different Node.js versions, and this plugin helps you handle those variations.

"eslint-plugin-promise": "^6.0.0"
This plugin provides ESLint rules for enforcing best practices and conventions related to JavaScript promises. Promises are used for handling asynchronous operations, and this plugin helps catch potential issues and enforce consistent usage.

"eslint-plugin-react": "^7.33.0"
This plugin provides ESLint rules specifically designed for React applications. It helps ensure best practices, detect potential issues, and enforce coding standards related to React components and JSX syntax.

"eslint-plugin-storybook": "^0.6.13"
This plugin is intended for projects that use Storybook, a popular tool for developing UI components in isolation. The plugin provides ESLint rules to ensure consistency and best practices when writing stories and configuring Storybook.

As discussed, will take note of the other plugins as well.